Vagrantfile

```diff
@@ -1,9 +1,20 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+unless Vagrant.has_plugin?("vagrant-service-manager")
+  raise <<-MSG
+    vagrant-service-manager plugin is required for projectatomic/adb.
+    Kindly install the plugin: `$ vagrant plugin install vagrant-service-manager`
+  MSG
+end
+
 Vagrant.configure(2) do |config|
+
   config.vm.box = "projectatomic/adb"
 
   config.vm.network "private_network", type: "dhcp"
 
+  # enable multiple providers as comma separated list. Eg: 'docker, openshift'
+  config.servicemanager.providers = 'docker'
```

Comment:
> Does this also mean that the docker certificates are only generated if the docker provider is set? They should really be generated regardless.

Need to check if the approach we are following (config + hook) can trigger task of configuring-docker without any mention of config in Vagrantfile explicitly.